content/en/docs/ibm/deploy/authentication.md

```diff
@@ -110,38 +110,165 @@ column):
+      name: kubeflow-gateway
+      namespace: kubeflow
     spec:
       selector:
         istio: ingressgateway
-    servers:
-    - hosts:
-      - '<hostname>'
-      port:
-        name: https
-        number: 443
-        protocol: HTTPS
-      tls:
-        mode: SIMPLE
-        privateKey: /etc/istio/ingressgateway-certs/tls.key
-        serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
-    ```
-
-8. Verify that the traffic is routed via HTTPS by using the value of 
-above-mentioned `Hostname` in your browser. It should redirect traffic from an 
+      servers:
+      - hosts:
+        - '<hostname>'
+        port:
+          name: https
+          number: 443
+          protocol: HTTPS
+        tls:
+          mode: SIMPLE
+          privateKey: /etc/istio/ingressgateway-certs/tls.key
+          serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
+    ```
+
+8. Verify that the traffic is routed via HTTPS by using the value of
+above-mentioned `Hostname` in your browser. It should redirect traffic from an
 HTTP address to HTTPS address automatically.
 
-**Note**: The certificates for the NLB DNS host secret expire every **90** days. 
+**Note**: The certificates for the NLB DNS host secret expire every **90** days.
 The secret in the `default` namespace is automatically renewed by IBM Cloud
-Kubernetes Service 37 days before it expires. After this secret is updated, you 
-must manually copy it to the `istio-ingressgateway-certs` secret by repeating 
-commands in step 5 and 6.
\ No newline at end of file
+Kubernetes Service 37 days before it expires. After this secret is updated, you
+must manually copy it to the `istio-ingressgateway-certs` secret by repeating
+commands in step 5 and 6.
+
+## Optional - KFServing configuration
+
+With this HTTPS setup, additional changes are needed to get KFServing to work.
+
+1. First, update the Knative domain that is used for the KFServing routes to the
+hostname that you used when updating `kubeflow-gateway`.
+
+    ```shell
+    kubectl edit configmap config-domain -n knative-serving
+    ```
+
+    This will open your default text editor, and you will see something like:
+
+    ```YAML
+    apiVersion: v1
+    data:
+      _example: |
+        ################################
+        #                              #
+        #    EXAMPLE CONFIGURATION     #
+        #                              #
+        ################################
+        # ...
+        example.com: |
+    kind: ConfigMap
+    ...
+    ```
+
+    Add a line above the `_example` key with your hostname as the key and an empty string value.
+    Be sure to update `<hostname>`:
+
+    ```YAML
+    apiVersion: v1
+    data:
+      <hostname>: ""
+      _example: |
+       ...
+    kind: ConfigMap
+    ...
+    ```
+
+    After you save and exit, routes for your InferenceServices will use this new domain.
+
+2. Since the certificates provided by IBM only allow for a single level of domain name added
```

Comment:
Suggestion:

- Add "Cloud" to "IBM"

```suggestion
2. Since the certificates provided by IBM Cloud only allow for a single level of domain name added
```